reposcan/reposcan.py

```diff
@@ -110,11 +110,12 @@ def is_running(cls):
 class ResponseJson(dict): # pylint: disable=too-few-public-methods
     """Object used as API response to user, represented as JSON"""
     def __init__(self, msg, success=True):
+        super(ResponseJson, self).__init__()
         self['msg'] = msg
         self['success'] = success
 
     def __repr__(self):
-        return json.dumps({"success1": self.success, "msg2": self.msg})
+        return json.dumps({"success1": self['success'], "msg2": self['msg']})
```

Comment:
Isn't just ```return json.dumps(self)``` better/easier?

Reply:
Or we can remove the `__repr__` method completely, we don't call str() anymore.

Reply:
ugh, I'm an idiot - mim, yeah, should just be calling dumps on self. jdobes, I'll leave __repr__ in there, we're not using str(ResponseJson) now, but may want//need to.